.github/workflows/codeql-pr.yml

```
@@ -0,0 +1,41 @@
+name: "CodeQL PR"
+
+on:
+  pull_request:
+    branches: [ "main" ]
+    types: [ opened, synchronize, ready_for_review ]
+
+jobs:
+  analyze:
+    name: Analyze (${{ matrix.language }})
+    runs-on: kibana
+    if: |
+      contains(github.event.pull_request.labels.*.name, 'ci:codeql') &&
+      contains(fromJSON('["MEMBER", "COLLABORATOR", "CONTRIBUTOR"]'), github.event.pull_request.author_association)
+      github.event.pull_request.user.type != 'Bot' &&
+      github.event.pull_request.draft == false
```

Seems like we missed the and operator between the contains expression and the expression that checks if PR user is not a bot; 

```suggestion
 contains(github.event.pull_request.labels.*.name, 'ci:codeql') &&
      contains(fromJSON('["MEMBER", "COLLABORATOR", "CONTRIBUTOR"]'), github.event.pull_request.author_association) &&
      github.event.pull_request.user.type != 'Bot' &&
      github.event.pull_request.draft == false 

```